src/main/java/picard/vcf/UpdateVcfSequenceDictionary.java

```diff
@@ -89,7 +91,15 @@ protected int doWork() {
         if (CREATE_INDEX)
             builder.setOption(Options.INDEX_ON_THE_FLY);
 
-        final VariantContextWriter vcfWriter = builder.setOutputFile(OUTPUT).build();
+        // handle stdout
+        if ( OUTPUT.getPath().equals(STD_OUT) ) {
```

Comment:
@yfarjoun I like the code change suggestion but it would it would be tough to validate the output. I can bypass that check in my test.Thoughts?